hangman.rb

```diff
@@ -0,0 +1,149 @@
+
+class Hangman
+	def initialize
+
+		# Library of words
+		@words = ["orca", "cheese", "cake", "pumpkin", "sun", "apple", "cloud", "paint", "bird", "travel", "lollipop", "ada"]
```

Comment:
We hadn't learned about constants yet at this point - but if you went back and refactored this would be a good thing to update